src/components/Filter/FilterSelect.vue

```diff
@@ -132,7 +126,7 @@ export default {
   },
 
   methods: {
-    filter,
+    // filter,
     ...mapMutations([
```

Comment:
I could not track down this reference and would need to make sure i did not break something here.